### PR TITLE
#157183398 Added asset Condition model  and tests

### PR DESCRIPTION
**Pull request by Awinja-Andela**
#### What does this PR do?
This pull request introduces Asset condition model and tests
#### Description of Task to be completed?
This should be possible:
When adding an asset for the first time
When assigning an asset to a user
When the asset is returned

#### How should this be manually tested?
python manage.py test
#### Any background context you want to provide?
User could not be able to write notes on the condition of the asset.
#### What are the relevant pivotal tracker stories?
#157183398
#### Screenshots (if appropriate)
screen shot 2018-05-11 at 11 52 43 am

### CODE REVIEW

File: core/models/asset.py [hunk 1]
```diff
@@ -112,9 +116,13 @@ class Asset(models.Model):
                                     editable=False,
                                     null=True,
                                     on_delete=models.PROTECT)
-    model_number = models.ForeignKey(AssetModelNumber, null=True,
+    model_number = models.ForeignKey(AssetModelNumber,
+                                     null=True,
                                      on_delete=models.PROTECT)
     current_status = models.CharField(editable=False, max_length=50)
+    condition_description = models.CharField(editable=False,
```
Comment (egichuri) [inline core/models/asset.py]: can we call this `asset_condition`?

File: core/models/asset.py [hunk 2]
```diff
@@ -225,6 +233,25 @@ def save(self, *args, **kwargs):
         super(AllocationHistory, self).save(*args, **kwargs)
 
 
+class AssetCondition(models.Model):
+    asset = models.ForeignKey(Asset,
+                              to_field="serial_number",
+                              null=False,
+                              on_delete=models.PROTECT)
+
+    condition_description = models.CharField(max_length=50,
```
Comment (egichuri) [inline core/models/asset.py]: same here

File: core/models/asset.py [hunk 3]
```diff
@@ -25,6 +25,10 @@
     (CHECKIN, "Checkin"),
     (CHECKOUT, "Checkout"),
 )
+NEW = "Brand New"
+WORKING = "Working"
+ISSUES = "Issues"
+NOT_WORKING = "Not Working"
```
Comment (egichuri) [inline core/models/asset.py]: do we need these?

Comment (Awinja-Andela) [reply core/models/asset.py]: Nope. Removed.

**Pull request merged**